### DISCUSSION TITLE
invalid expression: unexpected keyword `module`

### DISCUSSION THREAD

**Discussion by jason-pezzimenti** [vlang/v#21985] [Questions and Answers]
Sorry for the dumb question; I'm really new to V. I stumbled upon this error while running the v file in VS Code, and was wondering if somebody could help me understand what I'm doing wrong?

main.v:
```
module main
module token

fn main() {
	new_token := token.Token.access_modifier
	println(new_token)
}
```

src/token/token.v:
```
pub struct Token {
	token_type TokenType
	content string
	index int
}

enum TokenType as u8 {
	keyword
	operator
	identifier
	value
	access_modifier
	action_name
	symbol
}
```

I thought I was importing the module the right way, but obviously I've done something wrong. Just can't figure out where I've gone wrong.

Jase

**Comment by jason-pezzimenti** [comment]
I figured it out.

In main.v, I replaced `module token` with `import token`, then in token.v, I added `module main` to the top of the file.

**Reply by JalonSolov** [reply]
If `token.v` is meant to be part of module `token`, it should have `module token` at the top, not `module main`.

`module main` is the default if you don't have a `module` line at all.
